### PR TITLE
WIP - Add test to App

### PR DESCRIPTION
**Pull request by foxwellm**
Added a render test

### CODE REVIEW

File: src/containers/App/App.test.js [hunk 1]
```diff
@@ -1,23 +1,54 @@
...
+    ReactDOM.render(
+      <Provider store={store}>
+        <BrowserRouter>
+          <App />
+        </BrowserRouter>
+      </Provider>
+      , div);
+    ReactDOM.unmountComponentAtNode(div);
+  });
+
```
Comment (geet084) [inline src/containers/App/App.test.js:35]: I like that you could make that work and added it back in

**Review by geet084 — APPROVED** [review]

File: src/containers/App/App.test.js [hunk 2]
```diff
@@ -1,23 +1,54 @@
...
+    let mockProps2 = {
+      notes: [{ id: 1, title: 'Note Title' }],
+      items: [],
+      error: 'Something went wrong',
+      fetchNotes: jest.fn()
+    }
+    wrapper = shallow(<App {...mockProps2} />)
+    expect(wrapper).toMatchSnapshot()
+  })
+
```
Comment (ericweissman) [inline src/containers/App/App.test.js:51]: Thanks for adding a snaphot for the edge case of an error!

**Review by ericweissman — APPROVED** [review]
